addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/RFXrec433.xml

```diff
@@ -11,6 +11,11 @@
 				<label>Serial number</label>
 				<description>Serial number of the RFXCOM (FTDI) device</description>
 			</parameter>
+			<parameter name="disableDiscovery" type="boolean" required="true">
+				<label>Disable discovery of unknown devices</label>
+				<description>These RF protocols are prone to noise. If you find a lot of unknown devices showing up in your inbox enabling this will stop devices being added to your inbox.</description>
+				<default>true</default>
```

Comment:
I would expect the default to be false, or at least to have discovery enabled by default, otherwise we lose a valuable default setting.